docs/deployments/environments.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Environments / Instances
+description: An overview of the differences between Development and Production instances
+---
+
+# Environments / Instances
+
+When creating a new application within Clerk, you are typically provided with two instances: `Development` and `Production`.  These instances vary slightly and should be used appropriately.
+
+## Development Instance
+A `Development` instance is our default instanced type and has characteristics that allow it to be more useful for local development, and therefore has a more relaxed overall security posture.
+
+Some notable examples of `Development`-only characteristics are: 
+- A "Development" banner is shown prominently to make clear you're managing or configuring non-production data
+- Email and SMS templates are prepended with the environment type to prevent against using Development instances for production purposes
+- Some social connections use shared credentials by default 
+- Certain functionality like Bot Protection is only available in Production mode
```

Comment:
```suggestion
- Certain functionality like Bot Protection is only available in Production mode
- The Account Portal will use a Clerk development domain that ends with `accounts.dev` instead of your production domain
- OAuth consent screens will show the development domain that ends with `accounts.dev` instead of your production domain
- Search engines will not be able to crawl and index your application
```